pkgs/development/tools/build-managers/sbt/default.nix

```diff
@@ -15,6 +15,7 @@ stdenv.mkDerivation rec {
     mkdir -p $out/bin
     cat > $out/bin/sbt << EOF
     #! ${stdenv.shell}
+    export JAVA_HOME "${jre.home}"
```

Comment:
That syntax doesn't look right. Shouldn't this be `export "JAVA_HOME=$jre.home"`?

Also, please don't overwrite the setting of `JAVA_HOME` provided by the user's shell environment if there is some.